src/main/resources/target/checkstyle-cachefile

```diff
@@ -0,0 +1,2 @@
+#Thu Mar 28 15:03:39 EET 2024
+configuration*?=5CCF5BF28A362A244DDF2289713790EE3294547B
```

Comment:
What is this?

Reply:
> What is this?

I have no idea. this happened after I accidentally change resource folder.